userspace/falco/falco.cpp

```diff
@@ -1028,7 +664,7 @@ int falco_init(int argc, char **argv)
 
 		std::list<sinsp_plugin::info> infos = sinsp_plugin::plugin_infos(inspector);
 
-		if(list_plugins)
+		if(app.copts().list_plugins)
```

Comment:
As for the comment above, this would be a good fit for a `plugin_manager` class in the future (maybe in future PRs).

Reply:
Agreed!